src/components/list/list.tsx

```diff
@@ -170,6 +174,11 @@ export class List {
     };
 
     private setupList = () => {
+        if (this.mdcList) {
+            this.teardown();
```

Comment:
NICE! I was just thinking, I wonder if we just create a bunch of new listeners all the time, without removing the old ones, and then what do I see? Excellence, that's what! 👌 😄